pkg/tailscale/callbacks.go

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package main
+
+import (
+	"unsafe"
+
+	jnipkg "github.com/tailscale/tailscale-android/pkg/jni"
+)
+
+// #include <jni.h>
+import "C"
+
+var (
```

Comment:
sounds good. I think a lot of these will just go away too when we switch to having kotlin call localapiclient to editprefs instead of having Android use a chan to send an event